attributes/_install.rb

```diff
@@ -10,39 +11,7 @@
     '/etc/splunk'
   end
 
-default['splunk']['package']['version'] = '6.6.0'
-default['splunk']['package']['build'] = '1c4f3bbe1aea'
+default['splunk']['package']['version'] = '6.6.2'
```

Comment:
Probably should bump to 6.6.3 -> https://www.splunk.com/view/SP-CAAAP3H

Reply:
31c34317a5b50233361e85cedc7509acb1129192